libs/ledger-live-common/src/families/bitcoin/__snapshots__/bridge.integration.test.ts.snap

```diff
@@ -23221,24 +23220,15 @@ exports[`litecoin currency bridge scanAccounts litecoin seed 1 1`] = `
     "derivationMode": "native_segwit",
     "freshAddress": "ltc1qx2wxzwmpg4m8tr9d7rharerxaqj50jkdasvxmx",
     "freshAddressPath": "84'/2'/0'/0/3",
-    "freshAddresses": [
-      {
-        "address": "ltc1qpx9fcj5qht64vymj469cfyun73tllcch7qeuwq",
-        "derivationPath": "84'/2'/0'/0/0",
-      },
-    ],
```

Comment:
the left side is actually correct (starred, freshAddresses, name, unitMagnitude : these fields are gone). you may need to regenerate the snapshot (it can be done locally)

```
pnpm common ci-test-integration
```

Reply:
ok I will do that, thanks 👍